ports/atmel-samd/supervisor/port.c

```diff
@@ -71,6 +71,7 @@
 #ifdef CIRCUITPY_GAMEPAD_TICKS
```

Comment:
Change this to `#if CIRCUITPY_GAMEPAD` (note that it's not `#ifdef`). I should have done this in my refactor.